tailwind.config.ts

```diff
@@ -1,7 +1,7 @@
 import type { Config } from 'tailwindcss';
-import { theme } from './theme';
+import { theme } from './public/theme.js';
```

Comment:
Yeah, that might make more sense. Exporting it from `main.ts` would be nice so that we could
```ts
import { theme } from '@viamrobotics/prime'

Reply:
Importing from `main` will trigger a `'WARNING: Multiple instances of PRIME being imported.'`, so I'm going to leave this alone for now.

If we cared enough, we should probably do something more like this:

```ts
import { defineElements, theme } from '@viamrobotics/prime'

await defineElements() // <- component registration happens here, rather than import
```